wiki/Tournaments/OZT/2018/en.md

```diff
@@ -0,0 +1,114 @@
+# osu! Z-Tournament 2018
+![osu!Z-Tournament 2018](logo.png)
+
+- The **osu! Z-Tournament 2018**, is a 4v4 international osu!standard tournament.
+- There are no rank limits, therefore allowing any player to join.
+- Score system: **Scorev2**
+- This tournament will hold up to **64 teams, with 6 players at minimum and 8 players at maximum in each team.**
+- **Team admission will be decided through their PP average**, which means that the teams with the highest PP average will take priority.
+- **Group stages will be held round robin style**, afterwards the **Bracket stages will be double elimination**
+- It is **required** that all players join the discord
+- This page will only cover the general rules of the tournament, **you are required to read the ruleset.**
+- Any further questions must be forwarded to **[BearZyo](https://osu.ppy.sh/u/6116759) or [Div](https://osu.ppy.sh/u/3751116)**
+
+## Links
+- [Discord](https://discord.gg/29RmHDQ)
+- [Stream](https://www.twitch.tv/ztournament)
+- [Spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vT24OMXj6iKV2XeAHADas4JtTGi1E_dAfxeiAMt4-j_smF62yWz92H03LUrLkLjltk9M6zHz7iQSHlk/pubhtml)
```

Comment:
Could you be a bit more descriptive on the link name? I looked at it, but I wasn't sure what it was supposed to be about exactly.